src/extension.ts

```diff
@@ -560,5 +539,72 @@ function startKubeProxy(): Promise<string> {
 				reject(new Error(data.toString()));
 			});
 		}
-	}); 
+	});
+}
+
+function determinePlatform(): void {
```

Comment:
The real goal is more to determine the command curl based on the detected platform than to determine platform.

Reply:
see other PR comment